iroh-gateway/src/client.rs

```diff
@@ -69,8 +66,8 @@ impl http_body::Body for PrettyStreamBody {
     }
 }
 
-impl Client {
-    pub fn new(rpc_client: &iroh_rpc_client::Client) -> Self {
+impl<T: ContentLoader + Clone + std::marker::Unpin> Client<T> {
```

Comment:
is `Clone` needed, I thought `ContentLoader` already required that

Reply:
You're right, fixed in latest update.